### ISSUE TITLE
Log filled with "Jade will stringify dates in ISO form after 2.0.0"

### ISSUE THREAD

**Issue by OmgImAlexis** [pugjs/pug#2028]
Is there any way to squash this error or at least a way to work out which line it's coming from.
I get 100 or so of this line my log as soon as I visit certain routes.

**Comment by ForbesLindesay** [comment]
This is caused by code like the following

```jade
- var val = new Date();
input(value=val)
```

This code currently produces something like the following:

```html
<input value="Fri Aug 07 2015 14:30:39 GMT+0100 (BST)"/>
```

but in 2.0.0 it will produce:

```html
<input value="2015-08-07T13:31:03.206Z"/>
```

You can force this new behaviour now via:

```jade
- var val = new Date();
input(value=val.toISOString())
```

or keep the old behaviour via:

```jade
- var val = new Date();
input(value=val.toString())
```

Currently there's no way to affect this in bulk.  The other alternative option would be to do `npm install jadejs/jade` in order to fetch the experimental 2.0.0 release, which has the new behaviour and won't log any warnings.

**Comment by OmgImAlexis** [comment]
Is there any way to find where or at least which files this effects without having to scan through every instance of ``Date()`` in all my jade files? I'd like to fix all of my projects at once before the 2.0.0 release.

**Comment by ForbesLindesay** [comment]
Sorry, there isn't any easy way to do that built in.  You could replace that line of runtime.js with `throw new Error` instead of `console.warn`.
